plugin-server/src/utils/ai-costs/index.ts

```diff
@@ -0,0 +1,18 @@
+import { costs as anthropicCosts } from './anthropic'
+import { costs as cohereCosts } from './cohere'
+import { costs as deepseekCosts } from './deepseek'
+import { costs as googleCosts } from './google'
+import { costs as mistralaiCosts } from './mistralai'
+import { costs as openaiCosts } from './openai'
+import { costs as perplexityCosts } from './perplexity'
+import type { ModelRow } from './types'
+
+export const costs: ModelRow[] = [
```

Comment:
two things:
1. All the nested folders with index.ts feels like a waste now. Why not juts have `deepseek.ts`?
2. If you are auto generating the whole thing, why not just put it all in one file making the whole thing simpler?